src/Auth0.php

```diff
@@ -544,6 +544,42 @@ public function setRefreshToken($refreshToken)
         return $this;
     }
 
+    /**
+     * Renews the access token and ID token using an existing refresh token.
```

Comment:
I would probably mention here that the `scope` needs to include `offline_access` for the refresh token to be generated.